src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -4690,6 +4690,60 @@ LogicalResult ONNXCustomOp::inferShapes(
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// CallOpOld
+//===----------------------------------------------------------------------===//
+/// Infer the output shape of the ONNXCallOp. This method is required by
+/// the shape inference interface.
+LogicalResult ONNXCallOpOld::inferShapes(
+    std::function<void(mlir::Region &)> doShapeInference) {
+  // getResult().setType(getOperand().getType());
+  return success();
+}
+void ONNXCallOpOld::build(mlir::OpBuilder &builder, mlir::OperationState &state,
+    mlir::FuncOp function, mlir::ValueRange inputs) {
+  state.addAttribute(
+      ONNXCallOpOld::getCalleeAttrName(), SymbolRefAttr::get(function));
+}
+
+LogicalResult ONNXCallOp::verifySymbolUses(SymbolTableCollection &symbolTable) {
```

Comment:
This is needed by MLIR somewhere .... I think because of the DeclareOpInterfaceMethods<SymbolUserOpInterface>]> trait specified in the .td file cor ONNXCallOp. I basically copied it from the MLIR code.

Reply:
got it